openprescribing/api/views_spending.py

```diff
@@ -198,6 +225,34 @@ def price_per_unit(request, format=None):
     return response
 
 
+def _get_concession_bnf_codes(date):
+    """
+    Return list of BNF codes to which NCSO price concessions were applied in
+    the given month.
+
+    Note that because concessions are defined at the product-pack level and BNF
+    codes refer to products it's possible that the same BNF code may be
+    returned multiple times.
```

Comment:
Why not add a `SELECT DISTINCT`?  Also, why not use the ORM?

Reply:
Because I copied it from the big fat query in `spending_utils.py` and the deleted the stuff I didn't need. But you're quite right, looking at it now it's pretty easy to do via the ORM.